src/Resources/Resources.Test/ScenarioTests/ResourceTestRunner.cs

```diff
@@ -43,7 +43,7 @@ protected ResourceTestRunner(ITestOutputHelper output)
                 .WithExtraRmModules(helper => new[]
                 {
                     helper.RMResourceModule,
-                    helper.GetRMModulePath("AzureRM.Monitor.psd1")
+                    helper.GetRMModulePath("AzureRM.Monitor.psd1"),
```

Comment:
Add `helper.GetRMModulePath("AzureRM.Storage.psd1"),` here to use cmdlets in Storage when running test cases.

Reply:
@wyunchi-ms I made the suggested changes, but now it fails with "Device not configured"